task_three_and_four/src/test/java/com/epam/webdriver/aleksandrgostev/driver/DriverManager.java

```diff
@@ -14,13 +16,26 @@ private DriverManager() {
     }
 
     public static WebDriver getDriver() {
-        if (driver.get() == null) {
-            driver.set(new ChromeDriver());
-
+        if (null == driver.get()) {
+//            //How to run test with Param=kkk via Intellij IDEA?
+//            System.out.println(System.getProperty("kkk"));
+            switch (System.getProperty("browser")) {
```

Comment:
Let's update System.getProperty("browser") to System.getProperty("browser", "chrome")